advanced_functionality/tensorflow_bring_your_own/container/cifar10/nginx.conf

```diff
@@ -0,0 +1,25 @@
+events {
+    # determines how many requests can simultaneously be served
+    # https://www.digitalocean.com/community/tutorials/how-to-optimize-nginx-configuration
+    # for more information
+    worker_connections 2048;
+}
+
+http {
+  server {
+    # configures the server to listen to the port 8080
+    # https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms-inference-code.html#your-algorithms-inference-code-container-response
+    listen 8080 deferred;
+
+    # redirects requests from SageMaker to TF Serving
+    location /invocations {
+      proxy_pass http://localhost:8501/v1/models/cifar10_model:predict;
```

Comment:
You can do later: explain how this line works.

Reply:
added